cmd/skaffold/app/cmd/flags.go

```diff
@@ -368,6 +368,14 @@ var FlagRegistry = []Flag{
 		FlagAddMethod: "BoolVar",
 		DefinedOn:     []string{"render"},
 	},
+	{
+		Name:          "suppress-logs",
+		Usage:         "Suppress logs for specified stages in pipeline (build, deploy, status-check)",
```

Comment:
Do we want to suppress them or shorten them?
Could it just be a "--verbose-logs" that's on by default? I feel it's a bit overkill to be able to change the logs for each stage independently.

Reply:
@dgageot I do think that shorten is probably a better word to use over suppress. As for being able to control each stage independently, do you think that it's overkill from a user's perspective or from an implementation perspective? If from a user's perspective, I think we could support them using `=none` or `=all` in order to simplify things. This would allow them to not have to deal with per stage settings

Reply:
I think it's overkill for the users. But I'm open to other opinions :-)

Reply:
@GoogleContainerTools/skaffold-team anyone have thoughts on this?

Reply:
We talked about this in the logging breakout yesterday and it does seem useful to be able to suppress logs from different phases when the phase is successful.  This would be helpful for the Cloud Run emulation situation where Skaffold is being used as the mechanism for local building and deploying, but ideally shouldn't be shown to the user.

Reply:
In that situation, we might want to show the build logs but hide the deploy and status checks

Reply:
Let's do that then